apps/api/src/models/ApplicationData.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+from enum import Enum
+from typing import Optional
+
+from pydantic import BaseModel, EmailStr, Field, HttpUrl
+
+from .utils import form_body
+
+
+class Decision(str, Enum):
+    ACCEPTED = "ACCEPTED"
+    WAITLISTED = "WAITLISTED"
+    REJECTED = "REJECTED"
+
+
+Review = tuple[datetime, str, Decision]
+
+
+@form_body
+class RawApplicationData(BaseModel):
+    first_name: str
+    last_name: str
+    email: EmailStr
+    pronouns: list[str]
+    ethnicity: str
+    is_18_older: bool
+    university: str
+    education_level: str
+    major: str
+    is_first_hackathon: bool
+    portfolio_link: Optional[HttpUrl]
+    linkedin_link: Optional[HttpUrl]
+    collaboration_question: Optional[str] = Field()
```

Comment:
Thought: will the frontend specify null or empty string for these?
Suggestion: as mentioned before, I think we should follow FastAPI's suggestion and annotate with `Union[T, None]` instead of using `Optional`.